src/Package/Test/Repl/CurrentDirectoryTest.cs

```diff
@@ -52,7 +53,7 @@ public class CurrentDirectoryTest : IDisposable {
         public void DefaultDirectoryTest() {
             string myDocs = Environment.GetFolderPath(Environment.SpecialFolder.Personal);
             string actual;
-            using (var script = new VsRHostScript()) {
+            using (var script = new VsRHostScript(_sessionProvider, _broker.BrokerConnector)) {
```

Comment:
We can create `VsRHostScript` in `CurrentDirectoryTest` constructor and dispose in its `Dispose`.